### DISCUSSION TITLE
Specific folder to upload on neocities

### DISCUSSION THREAD

**Discussion by tlustopro** [bcomnes/deploy-to-neocities#106] [Q&A] [answered]
Hi, how to deploy specific folder from my repo on neocities please?

**Comment by bcomnes** [comment, marked as answer]
See https://github.com/bcomnes/deploy-to-neocities#inputs

> dist_dir: The directory to deploy to [Neocities](https://neocities.org/). Default: public

**Reply by tlustopro** [reply]
Thanks! Is it also possible to specify target folder on my Neocities web structure?

**Reply by bcomnes** [reply]
Im sorry I don't understand the question. Do you have an example?

**Reply by tlustopro** [reply]
Sorry for my not clear question.

Im using this action for publishing test reports (artifacts from my action) on my neocities web. The problem is that on every run save artifacts in the same folder in my github repo. So when i deploy this folder on neocities it is overwritten because has same name as folder from previous run.
So i was thinking if is possible to specify where (meaning route or folder on neocities) my files will be deployed. Some input like `target_dir`. 
Am i clear?

**Reply by bcomnes** [reply]
Oh I see. The model for this action is that your website source lives in git, is built in an action in a repeatable/stateless way, and synced to neocities in full, so your idea doesn't quite line up with that design. 

You can fiddle with setting `cleanup` option to false and the `protected_files` option to support 'state' on neocities if you really want, but it's not recommended. In your use-case, you could set cleanup to false, and then change the name of the artifacts at build time so they don't overwrite each other. This action will ignore any other files on neocities when set up this way. 

In terms of deploying a 'sub-folder' to neocities, just replicate whatever folder structure exists on neocities in your `dist_dir`, and file will end up where the need to.
